VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionPostsAdapter.java

```diff
@@ -30,13 +29,12 @@
     @ColorInt
     private final int edx_utility_success_dark;
 
-    @Inject
-    public DiscussionPostsAdapter(Context context, IEdxEnvironment environment) {
-        super(context, R.layout.row_discussion_thread, environment);
-        edx_brand_primary_base = context.getResources().getColor(R.color.edx_brand_primary_base);
-        edx_grayscale_neutral_light = context.getResources().getColor(R.color.edx_grayscale_neutral_light);
-        edx_brand_secondary_dark = context.getResources().getColor(R.color.edx_brand_secondary_dark);
-        edx_utility_success_dark = context.getResources().getColor(R.color.edx_utility_success_dark);
+    public DiscussionPostsAdapter(Fragment fragment, IEdxEnvironment environment) {
+        super(fragment.getContext(), R.layout.row_discussion_thread, environment);
```

Comment:
Actually, since these adapters are all being initialized from the `onViewCreated()` callback, and aren't used anywhere else, I guess we can just leave them as it is, and remove the local variable references to them.